tests/samplers_tests/tpe_tests/test_sampler.py

```diff
@@ -0,0 +1,12 @@
+import optuna
+from optuna.samplers import TPESampler
+
+
+def test_hyperopt_parameters():
+    # type: () -> None
+
+    # We can instantiate `TPESampler` with the parameters returned
+    # by `hyperopt_parameters()` method.
```

Comment:
IMO, it would be helpful if you move this comment to the documentation of `TPESampler.hyperopt_parameters` method. Although the example shows the usage of the method, the explicit explanation will help the understanding of users.
https://github.com/pfnet/optuna/pull/445/files#diff-17a6d8d7a5256b7902c4d43f333e0c9fR460

Reply:
Make sense. Moved: https://github.com/pfnet/optuna/pull/445/commits/df4fce904c3261b460ad6631b833be31c0be72b5